api/openapi-spec/swagger.json

```diff
@@ -845,11 +862,21 @@
           "type": "string"
         },
         "status": {
-          "description": "Status of the condition, one of True, False, Unknown.",
+          "description": "Status of the condition, one of True, False, Unknown.\n\nPossible enum values:\n - `False`:\n - `True`:\n - `Unknown`:",
```

Comment:
I was reading this sentence again, not a native speaker but the sentence "Possible enum values" sounds like the enum is open rather than closed, is it just me @jpbetz?